core/geometry/src/core-geometry.ts

```diff
@@ -254,9 +254,9 @@ export * from "./polyface/RangeTree/Point3dArrayRangeTreeContext";
 export * from "./polyface/RangeTree/LineString3dRangeTreeContext";
 export * from "./polyface/RangeTree/PolyfaceRangeTreeContext";
 export * from "./polyface/TaggedNumericData";
+export * from "./test/GeometrySamples";
```

Comment:
I'm pretty sure this change is wrong. I don't think test shows up in the distributed package at all, so this will cause a build error that may not be visible until after release. Did you mean `./serialization`?

Reply:
`GeometrySamples` file was initially in the src directory but it was only used by tests so I moved it to the test dir and removed it from `core-geometry.ts`. However, I noticed this file is used outside of geomlibs by  `src/test/render/primitives/Primitives.test.ts`. Technically they shouldn't be using it because it's tagged as `@alpha` but since they did, I had to add it back `core-geometry.ts`. 
@dassaf4 what do you think?

Reply:
I copied/pasted the code but simplified it as much as possible based on the args they pass. I prefer to keep the same polyface they were using before.